lms/views/lti/deeplinking.py

```diff
@@ -1,19 +1,19 @@
 """
-Views for handling content item selection launches.
+Views for handling deep linking launches.
 
-A content item selection request is an LTI launch request (so it's a form
+A deeplinking request is an LTI launch request (so it's a form
 submission containing all the usual launch params, including the OAuth 1
 signature) that's used by LMS's during the assignment creation process.
 
 When an instructor creates a Hypothesis assignment in an LMS that supports
-content item selection the LMS launches our app in order for us to present an
+deeplinking the LMS launches our app in order for us to present an
```

Comment:
content item is the right wording in the next line. We send "content items" using the deeplinking API.